lib/models/shows.js

```diff
@@ -65,13 +63,16 @@ async function getShowDetails(showId) {
 
   const neededShowDetails = {
     id: show.id,
-    key: show.id,
     title: show.name,
     poster: `https://image.tmdb.org/t/p/w780${show.images.backdrops[0].file_path}`,
     rating: show.vote_average,
-    genres: show.genres.map((genre) => {
+    genres_id: show.genres.map((genre) => {
+      return genre.id;
+    }),
+    genres_name: show.genres.map((genre) => {
       return genre.name;
     }),
+    networks: show.networks,
```

Comment:
The naming convention is different to your other variables:

```suggestion
    genreIds: show.genres.map((genre) => {
      return genre.id;
    }),
    genreNames: show.genres.map((genre) => {
      return genre.name;
    }),
    networks: show.networks,
```